docs.rs failed to build cargo_metadata-0.23.0

Log output from [docs.rs:](https://docs.rs/crate/cargo_metadata/0.23.0/builds/2546094)

```
# rustc version
rustc 1.92.0-nightly (c8905eaa6 2025-09-28)# docs.rs version
docsrs 0.6.0 (f42173de 2025-09-12)# build log
[INFO] running `Command { std: "docker" "create" "-v" "/home/cratesfyi/workspace-builder/builds/cargo_metadata-0.23.0/target:/opt/rustwide/target:rw,Z" "-v" "/home/cratesfyi/workspace-builder/builds/cargo_metadata-0.23.0/source:/opt/rustwide/workdir:ro,Z" "-v" "/home/cratesfyi/workspace-builder/cargo-home:/opt/rustwide/cargo-home:ro,Z" "-v" "/home/cratesfyi/workspace-builder/rustup-home:/opt/rustwide/rustup-home:ro,Z" "-e" "SOURCE_DIR=/opt/rustwide/workdir" "-e" "CARGO_TARGET_DIR=/opt/rustwide/target" "-e" "DOCS_RS=1" "-e" "CARGO_HOME=/opt/rustwide/cargo-home" "-e" "RUSTUP_HOME=/opt/rustwide/rustup-home" "-w" "/opt/rustwide/workdir" "-m" "6442450944" "--cpus" "6" "--user" "1001:1001" "--network" "none" "ghcr.io/rust-lang/crates-build-env/linux@sha256:e90291280db7d1fac5b66fc6dad9f9662629e7365a55743daf9bdf73ebc4ea79" "/opt/rustwide/cargo-home/bin/cargo" "+nightly" "rustdoc" "--lib" "-Zrustdoc-map" "--all-features" "--config" "build.rustdocflags=[\"--cfg\", \"docsrs\", \"-Z\", \"unstable-options\", \"--emit=invocation-specific\", \"--resource-suffix\", \"-20250928-1.92.0-nightly-c8905eaa6\", \"--static-root-path\", \"/-/rustdoc.static/\", \"--cap-lints\", \"warn\", \"--extern-html-root-takes-precedence\"]" "--offline" "-Zunstable-options" "--config=doc.extern-map.registries.crates-io=\"https://docs.rs/{pkg_name}/{version}/x86_64-unknown-linux-gnu\"" "-Zrustdoc-scrape-examples" "-j6" "--target" "x86_64-unknown-linux-gnu", kill_on_drop: false }`
[INFO] [stderr] WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
[INFO] [stdout] 1a0695abcc8c9d0522f64b22c08a59ef2536523d61be05b30ab55e7abdb22a90
[INFO] running `Command { std: "docker" "start" "-a" "1a0695abcc8c9d0522f64b22c08a59ef2536523d61be05b30ab55e7abdb22a90", kill_on_drop: false }`
[INFO] [stderr] warning: target filter specified, but no targets matched; this is a no-op
[INFO] [stderr]  Documenting cargo_metadata v0.23.0 (/opt/rustwide/workdir)
[INFO] [stderr] error[E0557]: feature has been removed
[INFO] [stderr]  --> src/lib.rs:1:29
[INFO] [stderr]   |
[INFO] [stderr] 1 | #![cfg_attr(docsrs, feature(doc_auto_cfg))]
[INFO] [stderr]   |                             ^^^^^^^^^^^^ feature has been removed
[INFO] [stderr]   |
[INFO] [stderr]   = note: removed in 1.58.0; see <https://github.com/rust-lang/rust/pull/138907> for more information
[INFO] [stderr]   = note: merged into `doc_cfg`
[INFO] [stderr] 
[INFO] [stderr] error: Compilation failed, aborting rustdoc
[INFO] [stderr] 
[INFO] [stderr] For more information about this error, try `rustc --explain E0557`.
[INFO] [stderr] error: could not document `cargo_metadata`
[INFO] running `Command { std: "docker" "inspect" "1a0695abcc8c9d0522f64b22c08a59ef2536523d61be05b30ab55e7abdb22a90", kill_on_drop: false }`
[INFO] running `Command { std: "docker" "rm" "-f" "1a0695abcc8c9d0522f64b22c08a59ef2536523d61be05b30ab55e7abdb22a90", kill_on_drop: false }`
[INFO] [stdout] 1a0695abcc8c9d0522f64b22c08a59ef2536523d61be05b30ab55e7abdb22a90
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,4 +1,4 @@
-#![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg))]
 #![deny(missing_docs)]
 //! Structured access to the output of `cargo metadata` and `cargo --message-format=json`.
 //! Usually used from within a `cargo-*` executable

